src/address/use-add-address.tsx

```diff
@@ -8,6 +8,7 @@ import useAddresses from "./use-addresses"
 const defaultOpts = {
 	url: "/api/bigcommerce/address",
 	method: "POST",
+  base: window.location.host,
```

Comment:
Can we fix the spacing here?  I just want it to be consistent with the rest of the code.

Also, do we have any concerns if people use this server-side?  I wonder if in the future, how we can ensure that it works on both.  Maybe we should do a check to see where the library is being used, and include an smarter default?

Reply:
Spacing fixed 👍 

Nice point, I have eliminated the use of `window`